### ISSUE TITLE
A problem occurred while testing your program

### ISSUE THREAD

**Issue by 18208943110** [songrise/CLIP-Count#7]
lightning_lite.utilities.exceptions.MisconfigurationException: The provided lr scheduler `StepLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.
 
How to solve？

**Comment by songrise** [comment]
hi. `lightning_lite.utilities.exceptions.MisconfigurationException:`. The exception seems caused by "lightning_lite", which is different from "pytorch_lightning", Did you install the correct `pytorch_lightning==1.8.0` module?
